index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Dunk card display</title>
+		<link rel="stylesheet" type="text/css" href="quize.css">
+	</head>
+	<body>
+
+		<article1 class="box">
+			<section class="section1">
+				<h2>Yellow Dunk Cards</h2>
+			</section>
+
+			<section class="section2">
+
+				<img src="http://www.solidbackgrounds.com/images/125x125/125x125-banana-yellow-solid-color-background.jpg">
+
+				<p class="discription">
+					waistcoat single origin coffe
+					Waistcoat kitsch and single-origin coffee anddreamcatcher deep v,B hoodie echo park bushwick yr blog. Mixtape ethical tattooed dreamcatcher bicycle rights try-hard vegan, seitan plaid. Artisan mixtape austin street art.
+				</p>
+			</section>
+
+			<section class="section3">
+
+				<p class="lable"> Available
+				</p>
+				<p class="big">specifications</p>
```

Comment:
Requirement 6: The second section contains product specifications. (You'll have to change where your sections begin and end)
6-i: This section should have a `<header>` containing the word "Specifications". (Switch the `<p>` tag out for a `<header>`)
6-ii: This section should contain two block elements (currently you've got the text you'll want to be elements as within a single element, break it out into two elements as described below)
6-iii: The first block element specifies the size.
6-iv: The second block element specifies the weight.